lib/widgets/workspace.dart

```diff
@@ -75,6 +108,21 @@ class _FilesWorkspaceState extends State<FilesWorkspace> {
     }
   }
 
+  void _switchWorkspaceView() {
+    setState(() {
+      switch (controller.view) {
+        case WorkspaceView.table:
+          controller.view = WorkspaceView.grid;
+          break;
+        case WorkspaceView.grid:
+          controller.view = WorkspaceView.table;
+          break;
+        default:
+          break;
```

Comment:
this entire thing is not useful, might be better to remove it (default behaviour for default is break)